laa_crime_application_store_app/services/v1/notification_service.py

```diff
@@ -0,0 +1,64 @@
+from uuid import UUID
+
+import httpx
+import structlog
+from fastapi import Request
+from sqlalchemy.orm import Session
+
+from laa_crime_application_store_app.models.subscriber_schema import Subscriber
+
+logger = structlog.getLogger(__name__)
+
+
+class NotificationService:
+    @staticmethod
+    def subscribe(db: Session, subscriber_type: str, webhook_url: str):
+        existing_subscriber = (
+            db.query(Subscriber).filter(Subscriber.webhook_url == webhook_url).first()
+        )
+        if existing_subscriber is not None:
+            return existing_subscriber
+
+        new_subscriber = Subscriber(
+            subscriber_type=subscriber_type,
+            webhook_url=webhook_url,
+        )
+        db.add(new_subscriber)
+        db.commit()
+
+        return None
```

Comment:
I think this should be returning the newly created record rather than none.

Reply:
And then the code above should be adjusted to allow for that.

Reply:
Cool, have adjusted